packages/keymaps/src/browser/keybindings-widget.tsx

```diff
@@ -224,14 +224,14 @@ export class KeybindingWidget extends ReactWidget implements StatefulWidget {
                     if (chunkIndex !== 0) {
                         collection.push({ value: '+', match: false, key: false });
                     }
-                    const chunkMatches = queryItems.includes(matchChunks[chunkIndex].toLocaleLowerCase());
-                    match ||= chunkMatches;
+                    const indexOfTerm = searchTerms.indexOf(matchChunks[chunkIndex].toLocaleLowerCase());
+                    const chunkMatches = indexOfTerm > -1 ? !!searchTerms.splice(indexOfTerm, 1) : false;
```

Comment:
This line is a bit confusing. I don't believe that `!!anything.slice(any, arguments)` can ever be false, so it seems this is just for the side effect of deletion.

Reply:
Yes, it can neve be false in this case. Should I split the deletion from setting `chunkMatches`?

Reply:
Yeah, I think that would be a little clearer - two lines for two concepts (match -> true + don't match same thing twice. You could even call `searchTerms` something like `unmatchedSearchTerms` to make it clearer that once something has been matched, it doesn't belong.).

Reply:
Thanks!